packages/kamino-sdk/src/whirpools-client/programId.ts

```diff
@@ -1,9 +1,11 @@
-import { PublicKey } from "@solana/web3.js"
+import { PublicKey } from '@solana/web3.js';
 
 // Program ID passed with the cli --program-id flag when running the code generator. Do not edit, it will get overwritten.
```

Comment:
Should we enable ability to pass some arguments by using .env?

Reply:
I don't think it's neccessary. This is an NPM package (library), so the .env file and environment variables should be specified by the main application that is using this SDK. This way they can use whatever they want and use this new method to override the mainnet program ID, if needed.

Reply:
make sense